pkgs/development/libraries/libexif/default.nix

```diff
@@ -11,6 +11,11 @@ stdenv.mkDerivation rec {
     sha256 = "0mzndakdi816zcs13z7yzp7hj031p2dcyfq2p391r63d9z21jmy1";
   };
 
+  patches = [
+    ./CVE-2020-0196.patch
```

Comment:
I guess you mean [CVE-2020-0198](https://nvd.nist.gov/vuln/detail/CVE-2020-0198)?

Reply:
Yep, fixed.